redash/app.py

```diff
@@ -42,7 +42,7 @@ def create_app():
     app = Redash()
 
     # Check and update the cached version for use by the client
-    app.before_first_request(reset_new_version_status)
+    reset_new_version_status()
```

Comment:
Any changes needed here? If the previous conditions are not used, it will seem uncontrollable and will not have the original effect?

Reply:
<img width="1720" alt="image" src="https://github.com/getredash/redash/assets/46274612/b9896af8-c6e0-4d6f-a979-ca4c5c32c168">

I change the code here in the same way as you, and it will cause an error that cannot be queried.

Reply:
https://github.com/pallets/flask/issues/4605

Based on the advice given by [davidism](https://github.com/davidism), we should directly invoke it within the create_app function.

Reply:
Also, suggestions from Reddit. https://www.reddit.com/r/flask/comments/11sx1sx/before_first_request_deprecated/